apps/web/src/pages/Profile/ExperienceAndSkillsPage/components/AddExperienceDialog.tsx

```diff
@@ -0,0 +1,257 @@
+      icon: UserGroupIcon,
+      title: intl.formatMessage({
+        defaultMessage: "Volunteer and community experience",
+        id: "Rz7WtH",
+        description: "Title for community experience section",
+      }),
+      description: intl.formatMessage({
+        defaultMessage:
+          "Gained experience by being part of or giving back to a community? People learn skills from a wide range of experiences like volunteering or being part of non-profit organizations, Indigenous communities, or virtual collaborations. This could be anything from helping during events and ceremonies, DJ-ing at a friends wedding, to gaming and streaming.",
+        id: "muduUA",
+        description: "Description for community experience section",
+      }),
+      buttonText: intl.formatMessage({
+        defaultMessage: "Add community experience",
+        id: "AKRd34",
+        description: "Button text to add a community experience to the profile",
+      }),
+      buttonPath: `${paths.createCommunity(applicantId)}${applicationParam}`,
+    },
+
+    {
+      icon: LightBulbIcon,
+      title: intl.formatMessage({
+        defaultMessage: "Personal experience",
+        id: "wTFUPE",
+        description: "Title for personal experience section",
+      }),
+      description: intl.formatMessage({
+        defaultMessage:
+          "People are more than just education and work experiences. We want to make space for you to share your learning from other experiences.",
+        id: "El/lyl",
+        description: "Description for personal experience section",
+      }),
+      buttonText: intl.formatMessage({
+        defaultMessage: "Add personal experience",
+        id: "L6UnSl",
+        description: "Button text to add a personal experience to the profile",
+      }),
+      buttonPath: `${paths.createPersonal(applicantId)}${applicationParam}`,
+    },
+
+    {
+      icon: StarIcon,
+      title: intl.formatMessage({
+        defaultMessage: "Award",
+        id: "+ikQY0",
+        description: "Title for award section",
+      }),
+      description: intl.formatMessage({
+        defaultMessage:
+          "Did you get recognized for going above and beyond? There are many ways to get recognized, awards are just one of them. ",
+        id: "KmqF8d",
+        description: "Description for award section",
+      }),
+      buttonText: intl.formatMessage({
+        defaultMessage: "Add award",
+        id: "CSg8cH",
+        description: "Button text to add an award to the profile",
+      }),
+      buttonPath: `${paths.createAward(applicantId)}${applicationParam}`,
+    },
+  ];
+
+  const Footer = React.useMemo(
+    () => (
+      <Dialog.Close>
+        <IconButton color="purple" icon={ArrowLeftIcon}>
+          {intl.formatMessage({
+            defaultMessage: "Cancel and go back",
+            id: "tiF/jI",
+            description: "Close dialog button",
+          })}
+        </IconButton>
+      </Dialog.Close>
+    ),
+    [intl],
+  );
```

Comment:
I believe it should be safe to just render this directly in the footer.

Reply:
:+1:  [Factor in footer](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5701/commits/a7fb63dce3403659343747cd850931fafe65ca46)